saleor/graphql/order/filters.py

```diff
@@ -27,7 +28,13 @@
 
 def filter_payment_status(qs, _, value):
     if value:
-        qs = qs.filter(payments__is_active=True, payments__charge_status__in=value)
+        if ChargeStatus.FULLY_REFUNDED in value:
+            qs = qs.filter(
+                Q(payments__is_active=True, payments__charge_status__in=value)
```

Comment:
Is it correct?
You are going to filter by `ChargeStatus.FULLY_REFUNDED` and also by `payments__is_active=True, payments__charge_status__in=value`. But the filter `ChargeStatus.FULLY_REFUNDED` will return you all records, including the ones from the `payments__is_active=True, payments__charge_status__in=value`.
Maybe I am missing something, but I would say that `qs = qs.filter(payments__charge_status=ChargeStatus.FULLY_REFUNDED)` should be enough here

Reply:
`value` is a list that can look like this for example: `['fully-refunded', 'partially-refunded']` . That is why I do it this way so filter will return both orders `FULLY_REFUNDED` and others.